### DISCUSSION TITLE
Crashlytics - iOS installation

### DISCUSSION THREAD

**Discussion by Anytram** [invertase/react-native-firebase#5679] [Q&A]
Hello,
I followed steps from [https://rnfirebase.io/crashlytics/usage#installation] and unfortunately I get those errors when trying to build iOS version: 
<img width="550" alt="Screenshot 2021-09-02 at 13 26 09" src="https://user-images.githubusercontent.com/37418524/131835670-56774cda-a653-4bce-99f1-5225c2b3f876.png">

Android version is building normally but crashes right afer opening. 
I have those modules from react-native-firebase installed: 
    "@react-native-firebase/analytics": "^11.5.0",
    "@react-native-firebase/app": "^11.2.0",
    "@react-native-firebase/crashlytics": "^12.7.4",
    "@react-native-firebase/messaging": "^11.2.0"
My app is using expo bare workflow -     "expo": "~40.0.0"
"react-native": "~0.63.4".
I tried to reintegrate pods and deleted all .lock files - still get the same errors.  I also tried manually iOS installation.

**Comment by mikehardy** [comment]
Hi there! Thou shalt not mix and match versions :-) https://invertase.io/blog/react-native-firebase-versioning

Use 12.7.4 for all versions and that should clear up

**Reply by Anytram** [reply]
Hi I tried that fix and get following error... 
<img width="1741" alt="Screenshot 2021-09-02 at 21 14 34" src="https://user-images.githubusercontent.com/37418524/132182185-72837ea2-81eb-41bf-8a56-83d52646f8cf.png">

**Reply by mikehardy** [reply]
1) Please post text not images
2) you still have analytics at a wrong version? It appears you have not followed my advice completely. All firebase modules, all same version
3) You may be having difficulties with some combo of what Xcode thinks you have for VALID_ARCHS / EXCLUDED_ARCHS etc, examine:

a) https://github.com/invertase/react-native-firebase/blob/ddceb896bcf186bc4976a1b07e984c8a93c5cab6/tests/ios/Podfile#L54-L55
b) https://github.com/invertase/react-native-firebase/blob/ddceb896bcf186bc4976a1b07e984c8a93c5cab6/tests/package.json#L60

Fixing those for you is project and machine-specific, it is out of scope for the help we may offer here.

You may examine the script (with comments worth reading) showing how to fully integrate react-native-firebase from scratch, comparing it's results with your project may help https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh
